modules/personalize_taxonomy_context/js/personalize_taxonomy_context.js

```diff
@@ -0,0 +1,21 @@
+(function ($) {
+
+  Drupal.personalize = Drupal.personalize || {};
+  Drupal.personalize.visitor_context = Drupal.personalize.visitor_context || {};
+  Drupal.personalize.visitor_context.taxonomy_context = {
+    'getContext': function(enabled) {
+      var vocabularies = Drupal.settings.personalize_taxonomy_context.vocabularies, context_values = [], vocabulary_machine_name;
```

Comment:
I think context_values needs to be initialized to an empty object, i.e. `{}` instead of `[]`